merlin/examples/workflows/feature_demo/feature_demo.yaml

```diff
@@ -109,12 +109,13 @@ study:
           shell: /usr/bin/env python2
           task_queue: pyth2_hello
 
-    - name: shutdown all workers
+    - name: shutdown
       description: |
-          shutdown all workers
+          shutdown all workers attached to this queue
       run:
          cmd: |
-            exit $(MERLIN_STOP_WORKERS)
+            exit $(MERLIN_HARD_FAIL)
```

Comment:
Why do you want to shutdown only some of the workers?

Reply:
hard_fail will stop all the workers in that workflow, since they all look at that queue; stop_workers stops all workers everywhere. i've put a note in there

Reply:
Ok, but in general HARD_FAIL will only stop the workers for queues associated with the specific failed step. Is this correct?

Reply:
yes

Reply:
as an alternative, it could shutdown all workers associated with all the queues in that workflow, but that would involve getting information that lives outside of the individual step. it might also have the side effect of bleeding into other workflows' workers via the default queue.

right now it's easiest to just stop workers on that specific queue, which i think should encourage users to be conscious of which queue they want to shut down